client/extensions/woocommerce/app/settings/payments/payment-method-stripe.js

```diff
@@ -152,11 +169,26 @@ class PaymentMethodStripe extends Component {
 							value="no"
 							checked={ 'no' === method.settings.capture.value }
 							onChange={ this.onEditFieldHandler } />
-						<span>{ translate( 'Authorize the customers credit card but charge manually' ) }</span>
+						<span>{ translate( 'Authorize the customer\'s credit card but charge manually' ) }</span>
+					</FormLabel>
+				</FormFieldset>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Descriptor' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **44** times:
`translate( 'Descripton' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).